test/services/user/services/AdminUsers.test.js

```diff
@@ -353,6 +353,26 @@ describe('AdminUsersService', () => {
 		const testStudent = students.find((stud) => mockStudent.firstName === stud.firstName);
 		expect(testStudent.birthday).equals('01.01.2000');
 	});
+
+	it('does not allow student user creation if school is external', async () => {
+		const schoolService = app.service('/schools');
+		const serviceCreatedSchool = await schoolService.create(
+			{ name: 'test', ldapSchoolIdentifier: 'testId' },
+		);
+		const { _id: schoolId } = serviceCreatedSchool;
```

Comment:
please use testObjects.createTestSchool

Reply:
when using that helper the isExternal attribute wasn't available when defining etiher ldapSchoolIdentifier or source, that's why I used the service